docs/robots/warthog/tutorials_warthog.mdx

```diff
@@ -4,9 +4,935 @@ sidebar_position: 4
+
+3.  The [rqt_robot_steering plugin](http://wiki.ros.org/rqt_robot_steering).
+    Run the `rqt` command, and select **Plugins→Robot Tools→Robot Steering** from the top menu.
+
+Warthog uses [twist_mux](http://wiki.ros.org/twist_mux) to mix separate [geometry_msgs\Twist](http://docs.ros.org/api/geometry_msgs/html/msg/Twist.html) control channels into the `warthog_velocity_controller/cmd_vel` topic.
+
+Additional velocity channels can be defined in [twist_mux.yaml](https://github.com/warthog-cpr/warthog/blob/kinetic-devel/warthog_control/config/twist_mux.yaml), or can be piped into the lowest-priority `cmd_vel` topic.
+
+#### Odometry
+
+Warthog publishes odometry information on the `odometry/filtered` topic, as [nav_msgs/Odometry messages](http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html).
+These are generated by [ekf_localization_node](http://wiki.ros.org/robot_localization), which processes data from several sensor sources using an Extended Kalman filter (EKF).
+This includes data from the wheel encoders and IMU (if available).
+
+#### Diagnostics
+
+:::note
+
+Diagnostics are only applicable to real Warthog robots, not simulation.
+
+:::
+
+Warthog provides hardware and software system [diagnostics](http://wiki.ros.org/diagnostics) on the ROS standard `/diagnostics` topic.
+The best way to view these messages is using the [rqt_runtime_monitor](http://wiki.ros.org/rqt_runtime_monitor) plugin.
+Run the `rqt` command, and select **Plugins→Robot Tools→Runtime Monitor** from the top menu.
+
+The same information is also published as a [warthog_msgs\Status](https://docs.ros.org/en/api/warthog_msgs/html/msg/Status.html) message on the `Status` topic.
+
+From your desktop, you can also launch the standard RQT Robot Monitor. This will report the diagnostic output
+from Warthog's self-monitoring capabilities, as shown in the figure below.
+
+```
+rosrun rqt_robot_monitor rqt_robot_monitor
+```
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/warthog_images/rqt_robot_monitor.png"
+      width="600"
+    />
+    <figcaption>Robot Monitor</figcaption>
+  </figure>
+</center>
+
+### Driving Warthog {#driving-warthog}
+
+There are three ways to drive Warthog and each way will work on a physical Warthog robot as well as on a simulated Warthog.
+
+1.  Using the interactive remote controller in RViz. See [Simulating Warthog](#simulating-warthog).
+2.  Using the controller for teleoperation. See below.
+3.  Publishing ROS messages. See below.
+
+:::warning
+
+Warthog is capable of reaching high speeds.
+Careless driving can cause harm to the operator, bystanders, the robot, or other property.
+Always remain vigilant, ensure you have a clear line of sight to the robot, and operate the robot at safe speeds.
+We strongly recommend driving in normal (slow) mode first, and only enabling turbo in large, open areas that are free of people and obstacles.
```

Comment:
The Futaba joystick has a dial to change the maximum speed. The 'slow' and 'turbo' modes are confusing for Warthog.

Reply:
I updated the language here.